generators/workspace/index.js

```diff
@@ -93,6 +94,12 @@ class Generator extends Base {
       default: defaultConfig.skipNextStepsLog,
       description: 'Skip the log message with the next steps at the end'
     });
+    this.option('addWorkspaceRepos', {
+      type: Boolean,
+      default: defaultConfig.addWorkspaceRepos,
+      description: 'true if the workspace repos should be part of the dependencies (desktop build) otherwise false (build process)'
```

Comment:
What are workspace repos?

```suggestion
      description: 'States whether workspace repos should be part of the dependencies. Set to `true` for local development setup. Otherwise `false` for CI build process.'
```